pkg/api/controller.go

```diff
@@ -1842,6 +1843,7 @@ func (c *Controller) handleAPIErrorCallback(ctx context.Context, w http.Response
 	case errors.Is(err, graveler.ErrDirtyBranch),
 		errors.Is(err, graveler.ErrCommitMetaRangeDirtyBranch),
 		errors.Is(err, graveler.ErrInvalidValue),
+		errors.Is(err, validator.ErrInvalidValue),
```

Comment:
Having to do this all the time is a recipe for forgetting to do it occasionally.  Can we have a single base for all of our errors, and then have each ErrInvalidValue wrap The `base_error.ErrInvalidValue`?

Out of scope for this PR, but I think we can have a nice win for programmability here.